Makefile

```diff
@@ -172,3 +180,55 @@ gcb-signer-image: out/gcb-signer-image
 gcb-signer-push-image: gcb-signer-image
 	docker push $(REGISTRY)/kritis-gcb-signer:$(IMAGE_TAG)
 
+# Steps to create a new test cluster on an GCP project that has already had a Kritis setup.
+.PHONY: create-integration-local
+create-integration-local:
+	gcloud --project=$(GCP_PROJECT) container clusters describe $(TEST_CLUSTER) >/dev/null \
```

Comment:
does this deserve to be in own script file?

Reply:
I'm on the fence here - adding another level of indirection doesn't have a clear benefit for me, but I can do so if you disagree.